<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Computer Animation: Physical Simulations</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="controlsSim">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Controls</string>
        </property>
        <layout class="QGridLayout" name="gridLayout"/>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="controlsScene">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Scene settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>No scene loaded</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="GLWidget" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="actionSceneBase"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionCameraReset"/>
    <addaction name="actionCameraX"/>
    <addaction name="actionCameraY"/>
    <addaction name="actionCameraZ"/>
   </widget>
   <addaction name="menuScene"/>
   <addaction name="menuCamera"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSceneBase">
   <property name="text">
    <string>Base</string>
   </property>
  </action>
  <action name="actionCameraReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionSceneProjectiles">
   <property name="text">
    <string>Projectiles</string>
   </property>
  </action>
  <action name="actionCameraX">
   <property name="text">
    <string>Align +X</string>
   </property>
  </action>
  <action name="actionCameraY">
   <property name="text">
    <string>Align +Y</string>
   </property>
  </action>
  <action name="actionCameraZ">
   <property name="text">
    <string>Align +Z</string>
   </property>
  </action>
  <action name="actionSceneCloth">
   <property name="text">
    <string>Cloth</string>
   </property>
  </action>
  <action name="actionSceneTestIntegrators">
   <property name="text">
    <string>Test: integrators</string>
   </property>
  </action>
  <action name="actionSceneTestColliders">
   <property name="text">
    <string>Test: colliders</string>
   </property>
  </action>
  <action name="actionSceneRope">
   <property name="text">
    <string>Elastic rope</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>code/glwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
